package.json

```diff
@@ -8,18 +8,22 @@
     "start": "node server.js",
     "lint": "eslint --ext .js,.json .",
     "test": "NODE_ENV=test mocha --recursive test",
-    "watch-dev": "nodemon server.js",
+    "watch-dev": "nodemon server.js | ./node_modules/.bin/bunyan",
     "watch-lint": "esw --watch .",
     "watch-test": "npm run test -- --watch --reporter min",
     "istanbul": "NODE_ENV=test istanbul",
     "generate-coverage": "npm run istanbul -- cover _mocha -- --recursive",
     "check-coverage": "istanbul check-coverage --config .istanbul.yml",
-    "upload-coverage-coveralls": "cat ./coverage/lcov.info | coveralls"
+    "upload-coverage-coveralls": "cat ./coverage/lcov.info | coveralls",
+    "snyk": "snyk",
+    "snyk-test": "snyk test --quiet",
+    "prepublish": "npm run snyk -- protect",
+    "git-hook": "npm run lint && npm run snyk-test && npm run generate-coverage && npm run check-coverage"
   },
   "config": {
     "ghooks": {
-      "pre-commit": "npm run lint && npm run generate-coverage && npm run check-coverage",
-      "post-rewrite": "npm run lint && npm run generate-coverage && npm run check-coverage"
+      "pre-commit": "npm run git-hook",
```

Comment:
I like this - some of our hooks are getting quite big and are used in several places.